MAINTAINING.md

```diff
@@ -0,0 +1,76 @@
+
+2.1 Check for code duplication  
+2.2 Check memory usage  
+2.3 Check all code paths  
+2.4 Check for API compliance  
+2.5 Check for consistent error handling  
+2.6 Check scope of variables and functions  
+2.7 Check for syntactical, semantical or logical errors  
+2.8 Check for any runtime efficiency improvements that can be made in specific lines of code - i.e., without changing the overall design of the code  
+
+
+### 3. - Test the PR
+
+Run tests to verify the correct behavior (see 1.6), both on Native and on a few selected boards, or present clearly and logically articulated reasons for skipping some/all tests.
+
+
+### 4. - Review the code against the coding conventions
+
+Check that the code follows the [Coding Conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions). This can be aided (but not replaced) by Uncrustify, using the uncrustify-riot.cfg file found in the root directory. Note the difference between personal coding style, which is subjective and is allowed subject to the other guidelines, and the coding conventions, which are absolute and must always be followed.
+
+
+### 5. - Review the documentation
+
+4.1 Check for sufficient high-level (module-level) documentation  
+4.2 Verify function level documentation  
+4.3 Are critical/hard to understand parts in the code documented?  
+4.5 Check grammar and spelling of documentation  
+
+
+## Non-technical guidelines
+
+### Interaction with contributors
+
+- Be responsive. Even if you are too busy to review the contribution, try to add a note fairly soon after the PR is submitted, thanking them for their awesome contribution and saying that you will review it in due course. Once the contributor has made their changes, ensure you reply to them in a reasonable timeframe. Acknowledge their replies to concerns if you're happy with their argument.
+- Be helpful. Give precise and correct advice when possible and when it will help the contributor. This can include code snippets, links to code/issues/wiki entries/web pages, or anything else. Educating contributors means we are investing in our community.
+- Be friendly. Be objective in your review, and of course, follow the [Code of Conduct](https://github.com/RIOT-OS/RIOT/blob/master/CODE_OF_CONDUCT.md).
+
+
+### Organisation of reviewing between maintainers
+
+This section is a placeholder for further discussion around making the review process more efficient and effective when multiple maintainers are involved. This could include:
+
+- Usage of labels
+- Division of review responsibilities, e.g. with ACKs for different areas of review
+- Usage of Github functionality re "Reviewers" and "Assignees"
```

Comment:
"**a**re"?

Reply:
re means "regarding/in the matter concerning" - https://en.oxforddictionaries.com/definition/re

Reply:
Have changed this for clarity